pokedex-react-routing/src/components/Home.js

```diff
@@ -36,11 +36,11 @@ export default class App extends Component {
     // Define handleSearch methods and pass them as props
     async handleSearch () {
         const response = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?pokemon=${this.state.searchQuery}`);
-        const pokemonData = response.results;
+        const pokemonData = response.body.results;
```

Comment:
to get things on the page, hit this with response.body.data.results to get your api connected to the original list!